src/mongo/mod.rs

```diff
@@ -18,6 +48,35 @@ impl Image for Mongo {
     fn ready_conditions(&self) -> Vec<WaitFor> {
         vec![WaitFor::message_on_stdout("Waiting for connections")]
     }
+
+    fn cmd(&self) -> impl IntoIterator<Item = impl Into<std::borrow::Cow<'_, str>>> {
+        match self.kind {
+            InstanceKind::Standalone => Vec::<String>::new(),
+            InstanceKind::ReplSet => vec!["--replSet".to_string(), "rs".to_string()],
+        }
+    }
+
+    fn exec_after_start(
+        &self,
+        _: testcontainers::core::ContainerState,
+    ) -> Result<Vec<ExecCommand>, testcontainers::TestcontainersError> {
+        match self.kind {
+            InstanceKind::Standalone => Ok(Default::default()),
+            InstanceKind::ReplSet => Ok(vec![ExecCommand::new(vec![
+                "mongosh".to_string(),
+                "--quiet".to_string(),
+                "--eval".to_string(),
+                "'rs.initiate()'".to_string(),
+            ])
+            .with_cmd_ready_condition(CmdWaitFor::message_on_stdout(
+                "Using a default configuration for the set",
+            ))
+            // Wait for the replica set to be ready
+            // This is a workaround because the replica set isn't immediately ready
+            // Without this delay, an immediate connection to the replica set would fail
+            .with_container_ready_conditions(vec![WaitFor::seconds(2)])]),
```

Comment:
Is there something better than hardcoded delay? (log, http call?)
Just wondering if we can make it more reliable

Reply:
You're right. Let me do some investing

Reply:
@DDtKey 
updated!

Reply:
And could you tell me how I should fix my PR name?

Reply:
I think there was a missing space, I've changed the naming a bit